google/cloud/storage/examples/storage_object_samples.cc

```diff
@@ -114,7 +114,14 @@ void InsertObjectStrictIdempotency(google::cloud::storage::Client unused,
   [](std::string bucket_name, std::string object_name, std::string contents) {
     // Create a client that only retries idempotent operations, the default is
     // to retry all operations.
-    gcs::Client client{gcs::ClientOptions(), gcs::StrictIdempotencyPolicy()};
+    auto status_or_credentials = gcs::oauth2::GoogleDefaultCredentials();
+    if (!status_or_credentials.ok()) {
```

Comment:
I am channeling @frankyn who prefers a blank line before and after the error handling code in these examples.  Also the `.ok()` should be removed here and elsewhere in these files.